search_config.install

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ *Implements hook_schema
+ */
+
+function search_config_schema() {
+  $schema['search_config_exclude'] = array(
+    'description' => 'Provides a way to exclude specific entities from the search results. Note that this does not stop the entities from being indexed.',
+    'fields' => array(
+      'entity_id' => array(
+        'description' => 'The {node}.nid to exclude.',
```

Comment:
the description should not refer node schema b/c entity id can vary for different content entities.